apps/test/unit/templates/sectionProgressV2/InviteToV2ProgressModalTest.jsx

```diff
@@ -0,0 +1,157 @@
+import {fireEvent, render, screen} from '@testing-library/react';
+import $ from 'jquery';
+import React from 'react';
+import sinon from 'sinon';
+
+import {UnconnectedInviteToV2ProgressModal} from '@cdo/apps/templates/sectionProgressV2/InviteToV2ProgressModal';
+import i18n from '@cdo/locale';
+
+import {expect} from '../../../util/reconfiguredChai';
+
+const DEFAULT_PROPS = {
+  setShowProgressTableV2: () => {},
+  setHasSeenProgressTableInvite: () => {},
+  setDateProgressTableInvitationDelayed: () => {},
+};
+
+describe('UnconnectedInviteToV2ProgressModal', () => {
+  let postStub;
+
+  beforeEach(() => {
+    postStub = sinon.stub($, 'post').returns(Promise.resolve({}));
+  });
+
+  afterEach(() => {
+    postStub.restore();
+  });
+
+  function renderDefault(propOverrides = {}) {
+    render(
+      <UnconnectedInviteToV2ProgressModal
+        {...DEFAULT_PROPS}
+        {...propOverrides}
+      />
+    );
+  }
+
+  it('renders the dialog with required elements', () => {
+    renderDefault();
+
+    expect(screen.getByText(i18n.progressTrackingAnnouncement())).to.exist;
```

Comment:
```suggestion
    screen.getByText(i18n.progressTrackingAnnouncement());
```
getByText will throw an error if it is not found. We don't need the `expect` statement and in fact get a much much better error message if we omit it.
Same for a bunch below.